src/bundle/Resources/translations/messages.en.xliff

```diff
@@ -433,6 +433,31 @@
         <target state="new">Show</target>
         <note>key: fieldview.toggler.show</note>
       </trans-unit>
+      <trans-unit id="3ed43e39f88eb44efe31d1befa4287bcf0501520" resname="focus_mode.disable.label">
+        <source>Default mode</source>
+        <target state="new">Default mode</target>
+        <note>key: focus_mode.disable.label</note>
+      </trans-unit>
+      <trans-unit id="4955a4170a73b14b10dd0387dc24e076ea6a09b3" resname="focus_mode.disable.tooltip">
+        <source>Exit focus mode.</source>
+        <target state="new">Exit focus mode.</target>
+        <note>key: focus_mode.disable.tooltip</note>
+      </trans-unit>
+      <trans-unit id="193e30b10643373d56daa5ee33b77e8f47d3f36d" resname="focus_mode.disable_hint">
+        <source>To exit focus mode, click the %icon% or press Esc.</source>
+        <target state="new">To exit focus mode, click the %icon% or press Esc.</target>
+        <note>key: focus_mode.disable_hint</note>
+      </trans-unit>
+      <trans-unit id="876c1f3845f42bec6ff26f25c8f2de53a4367539" resname="focus_mode.enable.label">
+        <source>Focus mode</source>
+        <target state="new">Focus mode</target>
+        <note>key: focus_mode.enable.label</note>
+      </trans-unit>
+      <trans-unit id="b4ddd4081d8dcbc38aa05ad83b0e79fbabd7cee0" resname="focus_mode.enable.tooltip">
+        <source>Enter focus mode. It shows only rich text editor toolbar and focuses your attention on the editing content.</source>
```

Comment:
Hi @ibexa/documentation, shouldn't `the` be removed from the sentence for it to read "... and focuses your attention on editing content"?

Reply:
It depends on the context, here we want to emphasise the content we edit, rather than on editing action.
We can leave it or remove, it still will be correct.

Reply:
I think I see your point. I the first case shouldn't it then say "and focuses your attention on the **edited** content"?

Reply:
Yes, your right, didn't notice that, my bad, thanks!
Let's change it a little:
"...focuses your attention on editing text."
What do you think?

Reply:
I like it. 🙂

Reply:
Great! So let's go with this version :)

Reply:
Done in https://github.com/ibexa/admin-ui/pull/955/commits/f0d08a7746a2971e4b8c5ac11da1f63f8dbade6b.